pkg/lifecycle/kustomize/kustomizer.go

```diff
@@ -230,8 +230,11 @@ func (l *Kustomizer) writeBase(step api.Kustomize) error {
 }
 
 func (l *Kustomizer) shouldAddFile(targetPath string) bool {
```

Comment:
I know this isn't really related to your change, but maybe this function should have a more explicit name, like `shouldAddFileToBase` or something

Reply:
updated to change the name